conf/application.conf

```diff
@@ -34,60 +56,63 @@ play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline'
 # ~~~~~
 # The secret key is used to secure cryptographics functions.
 # If you deploy your application to several instances be sure to use the same key!
-# Not set here so that MDTP frontends share the same secret key in the local environment
-# (see common.conf in frontend-bootstrap).
-# In server environments the secret comes from app-config-common
-# play.crypto.secret="gbSaMHOXet2Ma9P4N2hzrLyy4bsBaOUO6zFCzHm87RQTtVdVXLFVOQ7kfkYcSoLa"
-
-microservice {
-    metrics {
-        graphite {
-            host = localhost
-            port = 2003
-            prefix = play.${appName}.
-            enabled = true
-        }
-    }
+play.crypto.secret="UxhBwXnMmrL4fRKTZwcOduV2nrQhbilfTNZlCqyzR3Vb6VB3gQRjHrZqBd9ZPiMo"
```

Comment:
is this required ? 
looks like you copied it from some other service ?

Reply:
No its not, thanks, will remove